unity-ggjj/Assets/Scripts/SceneLoading/SceneLoader.cs

```diff
@@ -90,6 +93,9 @@ private IEnumerator LoadSceneCoroutine()
         SceneManager.UnloadSceneAsync(SceneManager.GetActiveScene());
     }
 
+    /// <summary>
+    /// Assigns the assigned narrative script to the NarrativeScriptStorage and starts the game
```

Comment:
Nit: At first glance "assigning an assigned script" was a bit difficult to understand. Would something like this work?
```suggestion
    /// Passes the narrative script of this SceneLoader instance to the NarrativeScriptStorage and starts the game
```